TESTING.md

```diff
@@ -169,8 +169,102 @@ cargo make \
   test
 ```
 
-(`-r` tells testsys to also show the status of resources like the cluster and instances in addition to tests):
-
 ```shell
 cargo make watch-test
 ```
+
+## Migration Testing
+
+Migration testing starts testsys at a starting version, or a provided initial image id and migrates instances to the target version.
+In order to accomplish this a few artifacts need to be created:
+* `TUF` repos that are publicly accessible
+* A previous release of bottlerocket signed with available keys
+* The ami id from the previous release
+* Build images and repos for current changes
+
+### The setup
+
+#### Prepare `Infra.toml`
+
+A publicly available s3 bucket is needed to complete the next steps, once a bucket is created the steps may be continued.
+`Infra.toml` is used by testsys to determine tuf repo locations, so `metadata_base_url` and `targets_base_url` need to be set based on the s3 bucket. The examples below assume `metadata_base_url` is the top level directory of the bucket and `targets_base_url` is a directory named `targets` in the top level directory. 
+The examples below also assume that the default repo in `Infra.toml`, but any repo can be used by setting the `PUBLISH_REPO` environment variable.
+
+#### Starting bottlerocket images
+
+In this example we will use `v1.8.0` as our starting bottlerocket version, but any tag from bottlerocket will work. 
+The following bash will checkout the proper branch from git and create the build images and repos for testing.
+
+```shell
+export TESTSYS_STARTING_VERSION="1.8.0"
+git checkout v${TESTSYS_STARTING_VERSION}
+cargo make
+cargo make ami
+cargo make repo
+```
+
+Next, the repos need to be added to the `TUF` repos (s3 bucket). Make sure to change `S3_BUCKET_NAME` to the `TUF` repo bucket name.
```

Comment:
I think all mentions of a S3 bucket should be removed, folks in the community may have their own way to update their TUF repos.

Reply:
See above comment. Should we remove the step by step guide so that we can remove the s3 bucket bits?